src/odemis/acq/stream/_base.py

```diff
@@ -681,8 +681,13 @@ def _update_linked_position(self, act, pos):
             # before updating va
             va.unsubscribe(self._axisvaupdaters[va_name])
             # update va
-            va.value = axpos
-            logging.info("Updating local axis %s to position %s", va_name, axpos)
+            axpos_clipped = va.clip(axpos)  # In case the axis went out of range
+            va.value = axpos_clipped
+            if axpos == axpos_clipped:
+                logging.info("Updating local axis %s to position %s", va_name, axpos)
+            else:
+                logging.warning("Updating local axis %s to clipped position %s (actual pos = %s)",
+                                va_name, axpos_clipped, axpos)
```

Comment:
Shouldn't the message be the other way around? e.g. if the axis position is the same as the axis position clipped -> Updating local axis %s to clipped position. Or do I have to read it differently?

Reply:
If the clipped value is the same are the original value, it means the "clipping" had no effect, which means it wasn't clipped. So we say it's been clipped if the clipped value is different from the original value. The code is correct.